airflow/exceptions.py

```diff
@@ -115,6 +115,17 @@ class AirflowClusterPolicyViolation(AirflowException):
     """Raise when there is a violation of a Cluster Policy in Dag definition"""
 
 
+class AirflowTimetableInvalid(AirflowException):
```

Comment:
```suggestion
class AirflowInvalidTimetable(AirflowException):
```

`AirflowTimetableInvalid` feels odd, no strong opinion though

Reply:
`AirflowTimetableInvalid` is more in line with other exceptions like `DagNotFound`, `DuplicateTaskIdFound` etc. (Naming of existing exceptions is already a bit inconsistent so I understand why you feel this way though.)